mkdocs.yml

```diff
@@ -11,6 +11,17 @@ nav:
     - 'Style Guidelines':
         - 'Go': 'style/go.md'
         - 'Markdown': 'style/markdown.md'
+    - 'Knowledge Base':
+        - 'Glossary': 'knowledge_base/glossary.md'
+        - 'Consensus':
+            - 'Avalanche':
```

Comment:
WDYT about changing this into Flare sub directory.

```
- Flare:
  - Consensus:
     - Introduction: ...
     - Avalanche: ...
     - Stellar: ...
```

Or replacing the order like:

```
- Avalanche:
  - Consensus: ...
- Stellar:
  - Consensus: ...
```

Reply:
You mean that Avalanche and Stellar logic will have to do with Flare only? In that case I would go with the first option
- Flare:
  - Consensus:
     - Introduction: ...
     - Avalanche: ...
     - Stellar: ...

Reply:
Yeah, that what I mean.

Reply:
I understand, then makes sense to have in Flare. Thanks!

Reply:
Please see here the reorganization https://github.com/optakt/optakt.github.io/pull/13/commits/6bc32ffc77ba3be455e377f190512bffc2ea752b